superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -185,13 +191,35 @@ function Welcome({ user, addDangerToast }: WelcomeProps) {
     'welcome.main.replacement',
   );
 
+  useEffect(() => {
+    const lastTab = bootstrapData.common?.conf.WELCOME_PAGE_LAST_TAB;
+    const [customTitle, customFilter] =
+      typeof lastTab === 'object' ? lastTab : [undefined, undefined];
```

Comment:
Thanks for catching this. Not sure what I've been thinking here other than I must have been thinking about the individual filters rather than the array containing the filters 🤦